src/Internal/Logger.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * Class Logger
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal;
+
+use WC_Logger;
+use WC_Payments;
+use WCPay\Core\Mode;
+use WCPay\Internal\Proxy\LegacyProxy;
+
+defined( 'ABSPATH' ) || exit; // block direct access.
+
+/**
+ * A wrapper class for interacting with WC_Logger.
+ */
+class Logger {
+
+	const LOG_FILENAME = 'woocommerce-payments';
+
+	/**
+	 * The holding property for our WC_Logger instance.
+	 *
+	 * @var WC_Logger $logger
+	 */
+	private $logger;
```

Comment:
Maybe naming this one `$wc_logger` would make it clearer what's going on?

Reply:
Done